pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -13,31 +13,15 @@
 
 let
 
-  inherit (pythonPackages) buildPythonPackage python isPy3k dbus-python enum34;
-
-  sip = if isPy3k then
-    pythonPackages.sip
-  else
-    (pythonPackages.sip_4.override { sip-module = "PyQt5.sip"; }).overridePythonAttrs(oldAttrs: {
-      # If we install sip in another folder, then we need to create a __init__.py as well
-      # if we want to be able to import it with Python 2.
-      # Python 3 could rely on it being an implicit namespace package, however,
-      # PyQt5 we made an explicit namespace package so sip should be as well.
-      postInstall = ''
-        cat << EOF > $out/${python.sitePackages}/PyQt5/__init__.py
-        from pkgutil import extend_path
-        __path__ = extend_path(__path__, __name__)
-        EOF
-      '';
-    });
+  inherit (pythonPackages) buildPythonPackage python isPy3k dbus-python sip pyqt-builder;
```

Comment:
The problem is that this package is using libsForQT5.callPackage https://github.com/NixOS/nixpkgs/blob/1c203a232544018680b757443083262a1bff8f20/pkgs/top-level/python-packages.nix#L6332

I thought it used the python callPackages.

Reply:
Would it be better if it used python callPackages?

Reply:
There is probably a reason why it is like this and I think it is to support multiple qt5 versions. If there is not a good reason it would be good if it would be converted.

Reply:
That way it only supports whatever version `libsForQt5` is, so I changed it to use python callPackage.